cluster-install/readme.adoc

```diff
@@ -129,6 +129,17 @@ ip-172-20-75-78.ec2.internal  node  True
 Your cluster example.cluster.k8s.local is ready
 ```
 
+if validations of the cluster can fail if the node-counts arent correct, it should infer sensible defaults (see #157). You can recreate the cluster specifying the counts.
```

Comment:
Wordings on the line "Sometimes the cluster creation can fail where only worker nodes are created and a master node is not provisioned. This is filed as https://github.com/kubernetes/kops/issues/3751[kops/#3751]. As a workaround, specifying the exact number of master node(s) and worker node(s) will create the cluster successfully".